src/coreComponents/constitutive/solid/CompressibleSolid.hpp

```diff
@@ -70,7 +74,7 @@ class CompressibleSolidUpdates : public CoupledSolidUpdates< NullModel, PORO_TYP
                                            real64 const & oldHydraulicAperture,
                                            real64 const & newHydraulicAperture ) const
   {
-    m_porosityUpdate.updateFromPressureAndTemperature( k, q, pressure, 0.0, 0.0, 0.0 );
+    m_porosityUpdate.updateFromPressureAndTemperature( k, q, pressure, 0.0, 0.0, 0.0, 0.0, 0.0 );
```

Comment:
you sure about zeros here, would "pressure" be more appropriate or it does not matter?

Reply:
I just followed the logic used before in this function because the zeros are not going to be used to update the porosity [here](https://github.com/GEOS-DEV/GEOS/blob/f45f89423c0020e471f88fc340cdc40524ff78ab/src/coreComponents/constitutive/solid/porosity/PressurePorosity.hpp#L52). But you are right, we should pass `pressure` here even if the function will not use it, it will be easier to understand and less error prone if someone ever changes the implementation of `PressurePorosity`.